### PR TITLE
Clean up SIG Governance

### PR DESCRIPTION
**Pull request by mrbobbytables**
This PR does 3 things, broken down by commit:
- bd5c10e8b5b217d108a9bf027f5db6a8018a5ce9 - This is a no-op, just general formatting and readability improvements.
- 2f4aafcf6892d5fb34bff81abf8f051def3f38c3 - Removes the requirement to send yearly sig updates to the mailing list. No SIG is actively doing this today.
- 14461a6fb57d89b58fb66328eac016a6cb3c81ea - Completely reworks the subproject section on creation and requirements to bring in line with how they are actually being managed.

/committee steering
/hold for review

### CODE REVIEW

**Review by BenTheElder — COMMENTED** [review]
First two commits LGTM, will circle back to the third sometime when the registry redirect is more settled.

File: committee-steering/governance/sig-governance.md [hunk 1]
```diff
@@ -201,64 +201,27 @@ Other roles...
 
 ### Subproject Creation
 
-#### Option 1: by SIG Technical Leads
-
-Subprojects may be created by [KEP] proposal and accepted by [lazy-consensus]
-with fallback on majority vote of SIG Technical Leads.  The result *SHOULD* be
-supported by the majority of SIG Leads.
-- KEP *MUST* establish subproject owners.
+Subprojects may be created with a majority vote of SIG Technical Leads.
```
Comment (tpepper) [inline committee-steering/governance/sig-governance.md:204]: Should we specify "simple majority"?

**Review by tpepper — COMMENTED** [review]
/lgmt

This is a pragmatic adjustment.

It might be beneficial to make explicit, but it's also probably sufficient that implied is a subproject might migrate to core or from core (this has happened obviously) and that this happens via the KEP process, with normal deprecation and alpha/beta/GA promotion policy also applying.

File: committee-steering/governance/sig-governance.md (hunk 1 above)
Comment (mrbobbytables) [reply committee-steering/governance/sig-governance.md:204]: sgtm, I'll update 👍

File: committee-steering/governance/sig-governance.md [hunk 2]
```diff
@@ -201,64 +201,27 @@ Other roles...
...
-Subprojects of the SIG *MUST* use the following processes unless explicitly
-following alternatives they have defined.
+Subprojects broadly fall into two categories, those that are directly part
+of [Kubernetes] core and those that are tool, driver, or other component that
+do not adhere to the Kubernetes release cycle.
 
-- Proposing and making decisions
-  - Proposals sent as [KEP] PRs and published to googlegroup as announcement
-  - Follow [KEP] decision making process
+**Kubernetes Core Subprojects**
```
Comment (BenTheElder) [inline committee-steering/governance/sig-governance.md:217]: For follow-up: is this well defined somewhere?

File: committee-steering/governance/sig-governance.md [hunk 3]
```diff
@@ -201,64 +201,27 @@ Other roles...
 
 ### Subproject Creation
 
-#### Option 1: by SIG Technical Leads
-
-Subprojects may be created by [KEP] proposal and accepted by [lazy-consensus]
-with fallback on majority vote of SIG Technical Leads.  The result *SHOULD* be
-supported by the majority of SIG Leads.
-- KEP *MUST* establish subproject owners.
+Subprojects may be created with a simple majority vote of SIG Technical Leads.
```
Comment (BenTheElder) [inline committee-steering/governance/sig-governance.md:204]: For follow-up: I'm not sure if this is consistently how SIGs handle this today, we may need to broadcast this to chairs&Tls and github admins.

**Review by BenTheElder — COMMENTED** [review]
/lgtm

**Pull request merged**